lib/composer/composer/installers/tests/Composer/Installers/Test/YawikInstallerTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+namespace Composer\Installers\Test;
+
+use Composer\Composer;
+use Composer\Installers\YawikInstaller;
+use Composer\Package\Package;
+use Composer\Package\PackageInterface;
+
+/**
+ * Class YawikInstallerTest
+ *
+ * @package Composer\Installers\Test
+ */
+class YawikInstallerTest extends TestCase
+{
+    /**
+     * @varComposer
+     */
+    private $composer;
+
+    /**
+     * @var PackageInterface
+     */
+    private $io;
+
+    /**
+     * @var Package
+     */
+    private $package;
+
+    /**
+     * setUp
+     *
+     * @return void
+     */
+    public function setUp()
+    {
+        $this->package = new Package('YawikCompanyRegistration', '1.0', '1.0');
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "YawikCompanyRegistration" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)